src/Akka.Persistence.Sql.HelperLib/TagTableMigrator.cs

```diff
@@ -52,10 +52,7 @@ public async Task Migrate(long startOffset, int batchSize, long? endOffset = nul
             await using var connection = _connectionFactory.GetConnection();
 
             // Create the tag table if it doesn't exist
-            var dbSchema = connection.GetSchema();
-
-            if (dbSchema.Tables.All(t => t.TableName != _journalConfig.TableConfig.TagTable.Name))
-                await connection.CreateTableAsync<JournalTagRow>();
+            await connection.CreateTableAsync<JournalTagRow>(TableOptions.CreateIfNotExists);
```

Comment:
TagTable name is inside the `TableOptions`, correct?

Reply:
No, `TagTable` name is inside the table mapping, this is moving the responsibility for detecting if the table exist or not from our code to the database.